### ISSUE TITLE
delete by api hidden dps

### ISSUE THREAD

**Issue by okletswin** [OpenTSDB/opentsdb#644]
while use /api/query to delete a metric, there always return the deleted rows in "dps" field.
some times it's to large for record the result
Can there provide a parameter for deleting quietly, and return the deleted count instead

**Comment by manolama** [comment]
Yeah, that's a good idea, wouldn't be difficult to implement. Got time? ;)

**Comment by okletswin** [comment]
@manolama OK, I'm expect~
